### DISCUSSION TITLE
New project ideas

### DISCUSSION THREAD

**Discussion by Adarsh-jaiss** [collab-community/.github#1] [Ideas]
-- @Abhinavkar  Do you have any project idea?

**Comment by KendallDoesCoding** [comment]
@Adarsh-jaiss I prefer we stick to project idea's in the discord, in the channel I made so community can all decide and discuss on if the idea is worth it or not.

**Comment by Abhinavkar** [comment]
> -- @Abhinavkar Do you have any project idea?

Yeah ! I have lots of ideas running through though

**Reply by KendallDoesCoding** [reply]
Please post it in the channel created on discord

**Comment by Abhinavkar** [comment]
- An open source website 
- A website for the solution of all leetcode problem 
- A website for collab community 
what are your views guys @everyone

**Reply by KendallDoesCoding** [reply]
> A website for the solution of all leetcode problem

@Abhinavkar Please can you create a repo for this and start work, we will help contribute in a bit!

**Comment by Abhinavkar** [comment]
Ya sure why not
